wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/AstToOutputsSectionElement.scala

```diff
@@ -1,13 +1,17 @@
 package wdl.draft3.transforms.ast2wdlom
 
+// TODO 2.11: Either.map doesn't exist in 2.11. When we're 2.12 only, remove this import of cats.syntax.either._
+import cats.syntax.either._
+
 import wdl.draft3.parser.WdlParser
 import wdl.model.draft3.elements.{OutputDeclarationElement, OutputsSectionElement}
 import wdl.draft3.transforms.ast2wdlom.EnhancedDraft3Ast._
 import common.Checked
 
-
 object AstToOutputsSectionElement {
   def convert(a: WdlParser.Ast): Checked[OutputsSectionElement] = {
-      a.getAttributeAsVector[OutputDeclarationElement]("outputs") map OutputsSectionElement
+      a.getAttributeAsVector[OutputDeclarationElement]("outputs") map { declarations =>
+        OutputsSectionElement(declarations)
```

Comment:
Is this line change still necessary when you add the import above?

Reply:
It is not! Proceeding with the import alone.